PhysicsTools/SelectorUtils/interface/PFJetIDSelectionFunctor.h

```diff
@@ -622,7 +647,8 @@ class PFJetIDSelectionFunctor : public Selector<pat::Jet> {
         passCut(ret, indexNNeutrals_FW_);
     }
 
-    else if ((version_ == SUMMER18PUPPI) || (version_ == RUN2ULPUPPI)) {
+    else if ((version_ == SUMMER18PUPPI) || (version_ == RUN2ULPUPPI) || (version_ = RUN3WINTER22PUPPIrunsBCDEprompt) ||
+             (version_ = RUN3WINTER22PUPPI)) {
```

Comment:
Typos to fix:
```suggestion
    else if ((version_ == SUMMER18PUPPI) || (version_ == RUN2ULPUPPI) || (version_ == RUN3WINTER22PUPPIrunsBCDEprompt) ||
             (version_ == RUN3WINTER22PUPPI)) {
```
(I suggest to always look at the outcomes of the static analyzer in the bot tests...)